scheduler/src/cook/kubernetes/api.clj

```diff
@@ -905,8 +909,8 @@
                    ;; preserving compatibility with Meosos + Cook Executor.
                    (not progress-file-path)
                    (assoc progress-file-var (str workdir "/" task-id ".progress")))
-        main-env-vars (make-filtered-env-vars main-env)
-        computed-mem (if checkpoint-memory-overhead (add-as-decimals mem checkpoint-memory-overhead) mem)]
+        computed-mem (if checkpoint-memory-overhead (add-as-decimals mem checkpoint-memory-overhead) mem)
+        main-env-vars (make-filtered-env-vars main-env computed-mem)]
```

Comment:
don't change the make-filtered-env-vars function, just move `computed-mem (if ...` above `main-env (cond-> main-env-base` and add computed-mem to main-env

Reply:
done